hub/src/server/http.ts

```diff
@@ -82,6 +82,37 @@ export function makeHttpServer(config: MakeHttpServerConfig & HubServerConfig &
       })
   })
 
+  app.delete(/^\/delete\/([a-zA-Z0-9]+)\/(.+)/, (
+    req: express.Request,
+    res: express.Response
+  ) => {
+    let filename = req.params[1]
+    if (filename.endsWith('/')){
+      filename = filename.substring(0, filename.length - 1)
+    }
+    const address = req.params[0]
+
+    server.handleDelete(address, filename, req.headers)
+      .then(() => {
+        res.writeHead(202)
+        res.end()
+      })
+      .catch((err: any) => {
```

Comment:
Can we have distinct error paths for:

* the file not existing?
* the path being invalid?
* the path not referring to something that can be deleted?

Reply:
Will do, I think this just requires creating a new error type and http response for file not existing.

Reply:
Fixed!

Reply:
Looks good to me!  Thanks for addressing this :+1: